streamer/src/nonblocking/quic.rs

```diff
@@ -702,17 +728,28 @@ async fn handle_connection(
     );
     let stable_id = connection.stable_id();
     stats.total_connections.fetch_add(1, Ordering::Relaxed);
+    let max_streams_per_100ms =
+        max_streams_for_connection_in_100ms(peer_type, params.stake, params.total_stake);
+    let mut last_throttling_instant = tokio::time::Instant::now();
+    let mut streams_in_current_interval = 0;
     while !stream_exit.load(Ordering::Relaxed) {
         if let Ok(stream) =
             tokio::time::timeout(WAIT_FOR_STREAM_TIMEOUT, connection.accept_uni()).await
         {
             match stream {
                 Ok(mut stream) => {
+                    if reset_throttling_params_if_needed(&mut last_throttling_instant) {
+                        streams_in_current_interval = 0;
+                    } else if streams_in_current_interval >= max_streams_per_100ms {
+                        let _ = stream.stop(VarInt::from_u32(0));
```

Comment:
Can we define a special stop code other 0 -- maybe too-busy to differentiate and for debug purpose?

Reply:
Suggest a code. I'll update the code.